st2auth_ldap_backend/ldap_backend.py

```diff
@@ -35,7 +36,8 @@ class LDAPAuthenticationBackend(object):
         * Bind Distinguish Name with user lookup.
         * Bind Distinguish Name with user and group lookup.
     """
-    def __init__(self, ldap_uri, use_tls=False, bind_dn='', bind_pw='', user=None, group=None):
+    def __init__(self, ldap_uri, use_tls=False, bind_dn='', bind_pw='', user=None, group=None,
+                 ref_hop_limit=0):
```

Comment:
Please update the configuration section in README to include this ref_hop_limit option. Also, I will merge https://github.com/StackStorm/st2-auth-backend-ldap/pull/8 before this PR. That PR will add an option to configure whether referrals should be chased. If option is set to chase referrals, then this option on hop limit makes sense.